helpers/round_changes.js

```diff
@@ -39,13 +39,13 @@ function RoundChanges(scope) {
 	if (exceptions.rounds[scope.round]) {
 		// Apply rewards factor
 		this.roundRewards.forEach((reward, index) => {
-			this.roundRewards[index] = new bignum(reward.toPrecision(15))
+			this.roundRewards[index] = new Bignum(reward.toPrecision(15))
```

Comment:
The reason why we have precision here to validate if the rewards exceed the total amount, which is if the rewards is too high then the `new Bignum(reward.toPrecision(15))` would become an exponential notation rather than fixed point.
```
var aa = new bignum(500000000)
aa.toPrecision(15) ====> '500000000.000000'
```
in case of exceeding 
```
var aa = new bignum(5000000000000000)
aa.toPrecision(15) ====> '5.00000000000000e+15'
```
conclusion: The precesion needs to exisits.